content/10-leetcode/00-code/00-py/two_pointer/lc_541_reverse_string.py

```
@@ -0,0 +1,18 @@
+import pytest
+
+def some_function() -> any:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)